### PR TITLE
fix create outing from route

### PR DESCRIPTION
**Pull request by ginold**
related to #959

`ngeoLocation.hasFragmentParam` stopped to work correctly, maybe some ngeo updates have changed the method?

### CODE REVIEW

**Comment by tsauerwein** [comment]
`ngeoLocation.hasFragmentParam` still should work fine. The problem is that the link (e.g. http://www.demov6.camptocamp.org/outings/add?r=822428) is wrong. The link should be `...#r=....`.

**Comment by ginold** [comment]
It is true that it works with `#` instead of `?`, however I like this code more than the previous.
Any opinion, @asaunier ?

**Comment by tsauerwein** [comment]
The reason we are using fragment parameters is that otherwise a new cache entry is created for each document. Fragment parameters are not send to the server, so the request to get the edit page will be the same for all documents of the same type. And this will one create one entry for each document type in the cache.

**Comment by asaunier** [comment]
> Any opinion, @asaunier ?

We have to be consistent and use ``#`` instead of ``?``.

**Comment by ginold** [comment]
ok! I replaced `?` with `#` and I'm keeping the code (still works fine and looks better).

File: c2corg_ui/static/js/outingediting.js [hunk 1]
```diff
@@ -69,10 +69,11 @@ app.OutingEditingController = function($scope, $element, $attrs, $http,
    */
   this.differentDates;
 
+
   // allow association only for a new outing to existing route
-  if (ngeoLocation.hasFragmentParam('r')) {
-    var urlParam = {'routes': ngeoLocation.getFragmentParam('r')};
-    appApi.getDocumentByIdAndDoctype(urlParam['routes'], 'r', appLang.getLang()).then(function(doc) {
+  if (ngeoLocation.getParam('r')) {
```
Comment (tsauerwein) [inline c2corg_ui/static/js/outingediting.js]: Shouldn't you be using `getFragmentParam`?

**Comment by ginold** [comment]
I corrected it in the second you wrote this comment :P

**Comment by ginold** [comment]
@asaunier  ok for merging?

File: c2corg_ui/static/js/outingediting.js [hunk 2]
```diff
@@ -69,10 +69,11 @@ app.OutingEditingController = function($scope, $element, $attrs, $http,
    */
   this.differentDates;
 
+
```
Comment (asaunier) [inline c2corg_ui/static/js/outingediting.js]: ?

**Pull request merged**